sdk/js/src/Client.ts

```diff
@@ -138,6 +137,84 @@ export class DVCClient<
         }
     }
 
+    /**
+     * Logic to initialize the client with the appropriate user and configuration data by making requests to DevCycle
+     * and loading from local storage. This either happens immediately on client initialization, or when the user is
+     * first identified (in deferred mode)
+     * @param initialUser
+     */
+    private clientInitialization = async (initialUser: DVCUser) => {
+        if (this.initializeTriggered || this._closing) {
+            return this
+        }
+        this.initializeTriggered = true
+
+        const storedAnonymousId = await this.store.loadAnonUserId()
+
+        this.user = new DVCPopulatedUser(
+            initialUser,
+            this.options,
+            undefined,
+            storedAnonymousId,
+        )
+
+        if (!this.options.disableConfigCache) {
+            await this.getConfigCache(this.user)
+        } else {
+            this.logger.info('Skipping config cache')
+        }
```

Comment:
we should probably put the log for `this.options.disableConfigCache` in `getConfigCache()`